controller/sentry/service.go

```diff
@@ -0,0 +1,51 @@
+package sentry
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/getsentry/sentry-go"
+	"github.com/giantswarm/microerror"
+	"github.com/giantswarm/micrologger"
+)
+
+type Config struct {
+	Dsn    string
+	Logger micrologger.Logger
+}
+
+type Service struct {
+	enabled bool
+	logger  micrologger.Logger
+}
+
+func New(config Config) (*Service, error) {
+	disabled := Service{enabled: false}
+	if config.Logger == nil {
+		return nil, microerror.Maskf(invalidConfigError, "%T.Logger must not be empty", config)
+	}
+	if config.Dsn == "" {
+		config.Logger.Log("level", "warning", "Sentry DSN is not set.")
```

Comment:
I understood this feature is optional. There should not be a warning log for optional features not being used. I would  actually drop the logs in here completely. We usually do not have logging for construction and initialization. Not sure how useful this is here.

Reply:
This could be useful to troubleshoot sentry integration itself.
If you think this should be working but you're not receiving errors in sentry, this message helps you understand the reason why. But I have no strong opinions on this.

Reply:
When I do not enable the optional feature I cannot expect it to work. So I do not want to be bothered with irrelevant information.

Reply:
removed